### PR TITLE
Reactome using utils

### PR DESCRIPTION
**Pull request by yoavkt**
Modify Reactome to use task retrieval

### CODE REVIEW

File: scripts/tasks_retrival/Reactome_tasks_creation.py [hunk 1]
```diff
@@ -90,33 +111,43 @@ def dump_to_task(task_dir, outcomes_df):
...
+        )
+
+    pathways_file = verify_source_of_data(
+        pathways_file, url=reactom_url, allow_downloads=allow_downloads
+    )
+    top_pathways_file = verify_source_of_data(
+        top_pathways_file, url=TOP_PATHWAYS_URL, allow_downloads=allow_downloads
+    )
+    df_path = pd.read_csv(pathways_file, index_col="Pathway identifier")
+
```
Comment (edenjenzohar) [inline scripts/tasks_retrival/Reactome_tasks_creation.py:138]: should `index_col` be i the header ?

Comment (yoavkt) [reply scripts/tasks_retrival/Reactome_tasks_creation.py:179]: I moved this into an external method. and now it is in the header

**Comment by yoavkt** [comment]
I added a print and verified that this works on local files as well.

**Comment by edenjenzohar** [comment]
I dont fully understand if these scripts have been checked for both of the possible instances: downloading the data and using local data. This is essential to make sure this script works correctly.

**Comment by yoavkt** [comment]
Ok, I removed the redundant script and removed it. I checked the Reactome_tasks_creation script with the addition of local and web-based tasks.

**Pull request merged**